backend/module/feedback/feedback.go

```diff
@@ -97,37 +97,39 @@ func (m *mod) GetSurveys(tx context.Context, req *feedbackv1.GetSurveysRequest)
 	// this scenario shouldn't happen as the request and config have to provide defined enum types
 	// TODO: if multiple orgins are requested at once and only some surveys are found, return partial error
 	if len(results) == 0 {
-		m.logger.Error("origins must be one of HEADER or WIZARD",
+		msg := "surveys not found for the requested origin(s)"
```

Comment:
another nit. original error msg was a lil misleading as they could have passed a valid origin but not have defined its survey questions in the config